src/charm_state.py

```diff
@@ -395,8 +395,10 @@ def check_reconcile_interval(cls, reconcile_interval: int) -> int:
         Returns:
             The validated reconcile_interval value.
         """
-        # The reconcile_interval should be greater than 0.
-        if reconcile_interval < 1:
+        # The reconcile_interval should be at least 2.
+        # Due to possible race condition with the message acknowledgement with job status checking
+        # in reactive process.
+        if reconcile_interval <= 1:
```

Comment:
The code indicates it should be "at least 1" ?

Reply:
The code should be if the reconcile_interval is 1 or less then raise error.

I think "at least 2" includes 2 right?

Reply:
You are right, as we cannot give fractions of minutes . I think `reconcile_interval < 2` would be more explicit in this case.